src/libraries/tests.proj

```diff
@@ -384,6 +384,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)\src\libraries\System.Buffers\tests\System.Buffers.Tests.csproj"
+                      Exclude="@ProjectExclusions" />
+    <!--
```

Comment:
This is just temporarily commented out, right?

Reply:
Yes, once the sanity check passes, I'll enable the rest.